### ISSUE TITLE
Is it possible to move the output panel to the right side?

### ISSUE THREAD

**Issue by catbaron-** [rgbkrk/atom-script#1113]
The output panel is in a class of "bottom", but I can't find a way to change the position of the output panel.
Any help?

**Comment by catbaron-** [comment]
Seems it is needed to hack the atom-message-panel package...

**Comment by nixel2007** [comment]
There is no built-in options for this, sorry :(

**Comment by mangueLinux** [comment]
Would love this too.

**Comment by ljacob15** [comment]
Agreed, this would make script much more user-friendly for those with wide screens.

**Comment by geniusupgrader** [comment]
Just referencing the pull request for that:
#1516

**Comment by piney** [comment]
Yep this would be a great idea, work around doesn't work for me or I'm just not doing it right

**Comment by GenericArt** [comment]
I too would love having the Output Panel on the side <3

**Comment by tiboine** [comment]
I second this.

**Comment by radicalCommit** [comment]
+1

**Comment by Mister7F** [comment]
+1

**Comment by MaxGraey** [comment]
https://github.com/rgbkrk/atom-script/pull/1516

**Comment by Singond** [comment]
The pull request #1516 does not provide an implementation of this. It only updates the version of the dependency in order to make such implementation possible. I am submitting an implementation in #2409, which makes it possible to configure the panel position. However, modifying the width (either interactively or by a setting) is still not possible, because the `atom-message-panel` package used for creating the panel does not support this.

For what it's worth, `atom-message-panel` is intended for displaying (probably short) messages, not program output. As a result, it's versatility is limited for our use-case: Fixed panel width and the inability to change the font size are the two insufficiencies which come to my mind now. Perhaps switching to another solution like output in a separate tab might be more user-friendly?
